### PR TITLE
Updating style guide

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: doc/developer/style-guide.md [hunk 1]
```diff
@@ -110,7 +112,7 @@ if (something)
 
 ### Strings
 
-- Use template strings instead of concatenation.
+- Use of template strings is perfered instead of concatenation.
```
Comment (jwilander) [inline doc/developer/style-guide.md]: "perfered" should be "preferred"

File: doc/developer/style-guide.md [hunk 2]
```diff
@@ -126,17 +128,18 @@ function wrongGetStr(stuff) {
...
 
-Part of the build process is running ESLint. ESLint is the final authority on all style issues. PRs will not be accepted unless there are no errors or warnings running ESLint. The ESLint configuration file can be found in: [web/react/.eslintrc](https://github.com/mattermost/platform/blob/master/web/react/.eslintrc.json)
+Part of the build process is running ESLint. ESLint is the final authority on all style issues. PRs will not be accepted unless there are no errors running ESLint. The ESLint configuration file can be found in: [web/react/.eslintrc](/web/react/.eslintrc)
 
 Instructions on how to use ESLint with your favourite editor can be found here: [http://eslint.org/docs/user-guide/integrations](http://eslint.org/docs/user-guide/integrations)
 
-This is an abridged version of the [Airbnb React/JSX Style Guide](https://github.com/airbnb/javascript/tree/master/react#airbnb-reactjsx-style-guide). Anything that is unclear here follow that guide. If there is a conflict, follow what is said below. 
+You can run eslint using the makefile by using `make check`
+
+The following is a subset of what ESLint checks for. ESLint is always the athority.
```
Comment (jwilander) [inline doc/developer/style-guide.md]: "athority" should be "authority"

**Comment by jwilander** [comment]
Other than the two spelling mistakes, it looks good

**Comment by crspeller** [comment]
Spelling fixed.

**Comment by hmhealey** [comment]
Looks good

**Pull request merged**